google/cloud/pubsub/samples/subscription_admin_samples.cc

```diff
@@ -109,12 +109,12 @@ void CreateSubscription(
 void CreateBigQuerySubscription(
     google::cloud::pubsub_admin::SubscriptionAdminClient client,
     std::vector<std::string> const& argv) {
+  // [START pubsub_create_bigquery_subscription]
```

Comment:
Won't that make the nesting unmatched?

Reply:
They should still match -- I'm adjusting it so the namespace lines are included

Reply:
Doesn't the `{` on line 117, now inside the snippet, match the `}` on line 135, outside the snippet?

Reply:
Ah, I see what you mean. It seems to be a bigger problem (in the whole file).

The code should be 
```
             << sub->DebugString() << "\n";
  }
//  [END pubsub_create_bigquery_subscription]
(std::move(client), argv.at(0), argv.at(1), argv.at(2), argv.at(3));
}
```